apps/admin_app/lib/admin_app_web/router.ex

```diff
@@ -94,6 +94,11 @@ defmodule AdminAppWeb.Router do
       ProductController,
       :delete_property
     )
+
+    # get("/shipping-policy", ShippingPolicyController, :index)
```

Comment:
@arjun289 can we remove the commented code?

Reply:
will remove this